app/models/spree/line_item.rb

```diff
@@ -193,6 +193,7 @@ def added_tax
       adjustments.tax.additional.sum(:amount)
     end
 
+    # Some of the tax rates may not be applicable depending to the order's tax zone
```

Comment:
I think that the method's name is misleading.
It returns all the tax rates regardless if they are application or not.

Reply:
Good point. I just did a quick search and found at least two usages of this method, and I wonder if this bug would need to be fixed there also?

- [checkout_helper.rb](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/helpers/checkout_helper.rb#L73-L75)
- [sales_tax_totals_by_producer.rb](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/lib/reporting/reports/sales_tax/sales_tax_totals_by_producer.rb#L25-L33)

Are you able to replicate the issue in checkout and the sales tax report? If so, we should raise an issue for that too, or update this one.